### PR TITLE
Fix Maven build (issue #2176)

### PR DESCRIPTION
**Pull request by msmuenchen**

### CODE REVIEW

File: src/main/java/org/bytedeco/javacv/FFmpegLogCallback.java [hunk 1]
```diff
@@ -41,7 +41,7 @@ public class FFmpegLogCallback extends LogCallback {
 
     static final FFmpegLogCallback instance = new FFmpegLogCallback().retainReference();
 
-    /** Returns an instance that can be used with {@link #setLogCallback(LogCallback)}. */
+    /** Returns an instance that can be used with {@link org.bytedeco.ffmpeg.global.avutil#setLogCallback(LogCallback)}. */
```
Comment (saudet) [inline src/main/java/org/bytedeco/javacv/FFmpegLogCallback.java:44]: Just `{@link avutil#setLogCallback(LogCallback)}` should work, no?

Comment (msmuenchen) [reply src/main/java/org/bytedeco/javacv/FFmpegLogCallback.java:44]: Nope, it doesn't:

```
/Users/xxx/Projects/javacv/src/main/java/org/bytedeco/javacv/FFmpegLogCallback.java:44: warning: Tag @link: reference not found: avutil#setLogCallback(LogCallback)
    /** Returns an instance that can be used with {@link avutil#setLogCallback(LogCallback)}. */
```

Comment (saudet) [reply src/main/java/org/bytedeco/javacv/FFmpegLogCallback.java:44]: Ok, anything else you might want to fix, or can we merge this?

Comment (msmuenchen) [reply src/main/java/org/bytedeco/javacv/FFmpegLogCallback.java:44]: I think we're good on that :)

**Review by saudet — APPROVED** [review]

**Pull request merged**